backend/shelf-server/src/main/java/com/htec/shelfserver/security/AuthorizationFilter.java

```diff
@@ -72,6 +85,11 @@ private UsernamePasswordAuthenticationToken getAuthentication(HttpServletRequest
         if (token != null) {
 
             token = token.replace(SecurityConstants.BEARER_TOKEN_PREFIX, "");
+
+            if (invalidJwtTokenRepository.findByJwt(token).isPresent()) {
+                throw new ShelfException();
```

Comment:
Maybe create supplier and shelf exception with some specific message?

Reply:
Because there is no HttpServletResponse in that method i have to throw an exception and catch it in the method above (doFilter) which has the response parameter. Also since I'm trying to send a response in middle of a filter I need to send it via HttpServletResponse